internal/rbd/nodeserver.go

```diff
@@ -101,6 +101,15 @@ var (
 	// xfsHasReflink is set by xfsSupportsReflink(), use the function when
 	// checking the support for reflink.
 	xfsHasReflink = xfsReflinkUnset
+
+	mkfsDefaultArgs = map[string][]string{
+		"ext4": {"-m0", "-Enodiscard,lazy_itable_init=1,lazy_journal_init=1"},
```

Comment:
having some references to these options and why we enable them as default options could help

Reply:
sure, but I don't know where they come from. This PR just moves them around...